cookbooks/chef-server-deploy/libraries/helpers.rb

```diff
@@ -14,6 +14,6 @@ def oc_id_applciation_config(app_name)
   end
 end
 
-Chef::Node.send(:include, ChefOpsDCC::Helpers)
-Chef::Recipe.send(:include, ChefOpsDCC::Helpers)
-Chef::Resource.send(:include, ChefOpsDCC::Helpers)
+Chef::Node.send(include ChefOpsDCC::Helpers)
+Chef::DSL::Recipe.send(include ChefOpsDCC::Helpers)
+Chef::Resource.send(include ChefOpsDCC::Helpers)
```

Comment:
```suggestion
Chef::Node.include ChefOpsDCC::Helpers
Chef::DSL::Recipe.include ChefOpsDCC::Helpers
Chef::Resource.include ChefOpsDCC::Helpers
```

This rubocop suggestion is misleading:

```
libraries/helpers.rb:17:12: W: Lint/SendWithMixinArgument: Use include ChefOpsDCC::Helpers instead of send(:include, ChefOpsDCC::Helpers).
Chef::Node.send(:include, ChefOpsDCC::Helpers)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
@phiggins Can you cut a PR to disable this on in cookstyle so we can avoid the confusion going forward?